src/components/opt-in-out/index.tsx

```diff
@@ -52,6 +52,18 @@ export default function OptInOut({ platform, redirectPath }: OptInOutProps) {
   function handleOptOut() {
     // @TODO - handle form submit
     Cookies.remove(PLATFORM_OPTIONS[platform].cookieKey)
+
+    // @TODO use zach's helper function for this from the video embed
+    // Ensures we don't send analytics data if the user hasn't consented
+    const hasConsentedAnalyticsTracking =
+      window && window.analytics && typeof window.analytics.track == 'function'
```

Comment:
@zchsh I stole this from your `checkAnalyticsConsent` function. If you merge your work in first, I'll refactor to use that util! Perhaps it would make sense to lift that up to the global `lib` now?